### ISSUE TITLE
Promote RSS icon/link on profile in web interface

### ISSUE THREAD

**Issue by Krinkle** [tootsuite/mastodon#14982]
# Pitch

Right now I find myself putting RSS feed links into my bio with an emoji, like so:

* <https://mastodon.technology/@krinkle>
* <https://polyglot.city/@krinkle>

<img width="500" alt="screen shot of public ui" src="https://user-images.githubusercontent.com/156867/95925785-9c25e100-0db2-11eb-891e-33bf255b24b3.png">

I'm proposing we add something like this (but perhaps not as prominent) to the "public web UI".

(Perhaps to the web app as well, in a menu somewhere, but it's far less important there since one wouid only be on those pages if you already have an account and are actively using it / logged-in to it).

### Motivation

I'd like to get more people into decentralised social media, and encourage more people to follow sites they find interesting directly, without being limited to site run by big companies and/or those who maintain dedicated FB/birdsite accounts for those blogs.

For people in a transitional state where they are on board with news readers, it can make it easier to embrace and feel comfortable, I t hink, if they find that the microblogs they counter (public toots on Mastodon) can be followed this way as well. At least, until they're ready to commit to Mastodon fully 😄 (which has a fairly high barrier to entry through picking an instance, creating an identity, and then navigating the still fairly immature landscape of mobile apps). Especially compared to the well-established, localised and mature reader apps out there.

I'd be willing to help write a patch if this change is welcomed.

**Comment by realpixelcode** [comment]
Good idea! I'd really like to have this feature as well.

**Comment by danrobi11** [comment]
Yes please. I came here to ask exactly this. Every Mastodon GUI frontend should have a RSS Feed icon link. @Krinkle Your icon looks good. Thanks for sharing.

**Comment by inson1** [comment]
Yes, please!
